src/routes/jira/workspaces/jira-workspaces-get.test.ts

```diff
@@ -0,0 +1,293 @@
+import { getLogger } from "config/logger";
+import express, { Application } from "express";
+import { Subscription } from "models/subscription";
+import { getFrontendApp } from "~/src/app";
+import supertest from "supertest";
+import { RepoSyncState } from "models/reposyncstate";
+import { Installation } from "models/installation";
+import { encodeSymmetric } from "atlassian-jwt";
+import { Errors } from "config/errors";
+import { DEFAULT_LIMIT, Workspace } from "routes/jira/workspaces/jira-workspaces-get";
+
+const createRepos = async(subscriptions: Subscription[]) => {
```

Comment:
For the test where we're passing a searchQuery param, I wanted a little more control over the repoOwner naming so created this function just for that use case. createRepositories is a generic function used to create 1 repo for each subscription (basically where we don't care what the repoOwner name is).